### DISCUSSION TITLE
iModel Bank service setup instructions?

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#4771] [Q&A] [answered]
I am looking for instructions or a sample to code up an iModelBank service.

**Comment by john-florence** [comment, marked as answer]
Hi John, the only way to interact with an iModel is via our iModels REST API or a client library supporting it. https://developer.bentley.com/apis/imodels-v2/

**Comment by JohnMajerle** [comment]
FYI - https://github.com/iTwin/itwinjs-core/discussions/4856
